packages/oui-modal/_mixins.less

```diff
@@ -92,6 +92,21 @@
       }
     }
 
+    &__loader {
+      position: absolute;
```

Comment:
You should precise `top` and `left` values to 0 :)

Reply:
yes done!